src/services/labellang/KeyLabelNeo2.ts

```diff
@@ -0,0 +1,203 @@
+import { KeyLabel, MOD } from './KeyLabel';
```

Comment:
@Silverquark The `MOD` is not used in this file, therefore the warning message is shown at building. Could you remove it if it is unnecessary?